### ISSUE TITLE
Variation in metric output 

### ISSUE THREAD

**Issue by sharma-ji** [catboost/catboost#1118]
Problem:
catboost version:
Operating System: Windows 10
CPU: AWS

Hi, I am using catboost for binary classification of imbalanced data. My eval metric is AUC ROC. I am running stratified KFold with 10 folds. Whenever I run the same script script other day, the verbose output of ROC while training is different from previous run. I am using random_seed in CatBoostClassifier, random_state in StratifiedKfold as well as randon_state in train_test_split. Still there is variation in verbose output.

Thanks

**Comment by annaveronika** [comment]
Duplicate of https://github.com/catboost/catboost/issues/1115

**Comment by sharma-ji** [comment]
I am not using GPU, its all training on CPU, so I believe my issue is different from the [#1115](https://github.com/catboost/catboost/issues/1115), if it is not same for CPU

**Comment by P7668** [comment]
@sharma-ji , you said, "I run the same script other day". I am in doubt that if you run your script on same day several times so are you getting same output ?

**Comment by sharma-ji** [comment]
Yes, for the same day I get same output whenever I re-run the script. I save the notebook and run next day, and I see variations in the verbose AUC outputs.

**Comment by annaveronika** [comment]
This should not happen, could you please check that you are using the latest version of the library?

**Comment by sharma-ji** [comment]
@annaveronika  `catboost.__version__` returns `0.17.5`

**Comment by P7668** [comment]
@sharma-ji, are you sure that you are not using any parameter which is taking current system date as value in your whole program.

**Comment by annaveronika** [comment]
@sharma-ji First of all, please update to the latest version. Secondly, I'm not sure why you have different results on every run, random seed is fixed starting from earlier versions, and after that all calculations are deterministic. There should be some problem on your side.

**Comment by annaveronika** [comment]
closing because of inactivity
